docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -8,133 +8,123 @@
 
 | `metrics-apm.service_destination_1m`
-| 7 days / 50GB
 | 90 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_destination_10m`
-| 14 days / 50GB
 | 180 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_destination_60m`
-| 30 days / 50GB
 | 390 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_summary_1m`
-| 7 days / 50GB
 | 90 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_summary_10m`
-| 14 days / 50GB
 | 180 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_summary_60m`
-| 30 days / 50GB
 | 390 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_transaction_1m`
-| 7 days / 50GB
 | 90 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_transaction_10m`
-| 14 days / 50GB
 | 180 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.service_transaction_60m`
-| 30 days / 50GB
 | 390 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.transaction_1m`
-| 7 days / 50GB
 | 90 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.transaction_10m`
-| 14 days / 50GB
 | 180 days
 | Aggregated transaction metrics powering the APM UI
 
 | `metrics-apm.transaction_60m`
-| 30 days / 50GB
 | 390 days
 | Aggregated transaction metrics powering the APM UI
 
 |===
 
-The APM index lifecycle policies can be viewed in {kib}.
-Navigate to *{stack-manage-app}* / *Index Lifecycle Management*, and search for `apm`.
+Rollover (writing to a new index) prevents a single index from growing too large and optimizes indexing and search performance.
+Rollover occurs after either an age or size metric is met.
+The default rollover definition for each APM data stream is applied based on {ref}/data-stream-lifecycle-settings.html#cluster-lifecycle-default-rollover[`cluster.lifecycle.default.rollover`].
+
+The APM data stream lifecycle policies can be viewed in {kib}:
+
+. Navigate to *{stack-manage-app}* → *Index Management* → *Component Templates*.
+. Search for `apm`.
+. Look for templates with `@lifecycle` suffix.
 
-TIP: Default {ilm-init} policies can change between minor versions.
-This is not considered a breaking change as index management should continually improve and adapt to new features.
+TIP: Default lifecycle policies can change between minor versions. This is not considered a breaking change as index management should continually improve and adapt to new features.
 
 [discrete]
 [id="apm-data-streams-custom-policy{append-legacy}"]
 == Configure a custom index lifecycle policy
 
-When the APM integration is installed, {fleet} creates a default `*@custom` component template for each data stream.
+The {es} apm-data plugin creates a default `*@custom` component template for each data stream.
```

Comment:
I wonder if we should revert this. AFAIK, the apm-data plugin doesn't actually create the custom component template but rather refers to then in the index templates. The creation part is mostly left to the users.

Reply:
Perhaps just reword slightly? Perhaps something along the lines of:

"Mappings and settings for data streams can be customised through the creation of `*@custom` component templates, which are referenced by the index templates created by the {es} apm-data plugin."